source/1_engine/rex_engine/src/diagnostics/logging/logger_config.cpp

```diff
@@ -114,11 +121,24 @@ namespace rexlog
           }
         }
 
-        details::Registry::instance().set_levels(rsl::move(levels), global_level_found ? &global_level : nullptr);
+        rexlog::details::Registry::instance().set_levels(rsl::move(levels), global_level_found ? &global_level : nullptr);
       }
 
     } // namespace helpers
-  }   // namespace cfg
-} // namespace rexlog
 
-// NOLINTEND(misc-definitions-in-headers)
+    // search for REXLOG_LEVEL= in the args and use it to init the levels
+    void load_log_levels(const rsl::vector<rsl::string_view>& args)
```

Comment:
I'd like to have a good way of getting all the possible commandline arguments for the engine, like you can do with python "--help" command.
it's outside of the scope of this change, but just to let you know that this code will change slightly to follow that design